wls-gui-wahllokalsystem/src/components/stimmabgabevermerke/TheUWBStimmabgabevermerkeErfassenTable.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <v-container v-if="stimmabgabevermerke">
+    <div class="d-flex">
+      <v-number-input
+        v-model="rowSize"
+        :rules="[required, minNumber(1), maxNumber(250)]"
+        max-width="15rem"
+        label="Anzahl der Blätter"
+      />
+      <v-btn
+        class="ml-4 mt-3"
+        primary
```

Comment:
denke das war verwechselt, aber primary hat keinen effect
```suggestion
        active
```